### PR TITLE
Facebook Open Graph Tags

### PR DESCRIPTION
**Pull request by shivamMg**
https://github.com/fossasia/open-event-orga-server/issues/2145

@niranjan94 Please review and merge.

### CODE REVIEW

File: app/__init__.py [hunk 1]
```diff
@@ -96,6 +97,9 @@ def create_app():
     app.config['SQLALCHEMY_RECORD_QUERIES'] = True
     # app.config['SERVER_NAME'] = 'http://127.0.0.1:8001'
     # app.config['SERVER_NAME'] = 'open-event-dev.herokuapp.com'
+
+    # Facebook APP ID
+    app.config['FB_APP_ID'] = os.getenv('FB_APP_ID', '')
```
Comment (niranjan94) [inline app/__init__.py]: No. Don't set it here. Allow the admin to change it at `/admin/settings/`.

Comment (shivamMg) [inline app/__init__.py]: Turns out it's App ID is same as Facebook Client ID in Settings model.

**Comment by niranjan94** [comment]
@shivamMg also add the same Facebook App ID field along with the social links on the super admin settings panel.

**Comment by mariobehling** [comment]
In the light, that we still have a lot of bugs and basics not working I do not understand why we need to spend time on this kind of social media issue.

**Comment by shivamMg** [comment]
will submit another PR.

**Pull request closed without merging**